src/test/regress/expected/isolation_metadata_sync_vs_all.out

```diff
@@ -388,30 +388,6 @@ create_distributed_table
 step s2-commit:
  COMMIT;
 
-step s3-compare-snapshot:
```

Comment:
Would it help for ordering if we do `SELECT unnest(activate_node_snapshot() ORDER BY 1)`? I'm fine to skip, but looks like a useful test.

Reply:
no, it's not the ordering of the commands that is the issue, but the ordering of the command that inserts record into pg_dist_object. It is in dependency order, but independent objects are not ordered. They would have to be ordered by name, since OIDs / ObjectAddress are not synced. It seems like more trouble than it's worth for this one quirky test.